radiation/python/radsw/radsw_main.py

```diff
@@ -0,0 +1,4031 @@
+import numpy as np
+import xarray as xr
+import os
+import sys
+
+sys.path.insert(0, "/Users/AndrewP/Documents/work/physics_standalone/radiation/python")
```

Comment:
we'll just need to take a swing at some point and replace the hard-coded paths, but no need for this PR to do that.

Reply:
Yup will do soon